src/components/messenger/MessageBox.vue

```diff
@@ -264,7 +291,7 @@ export default {
 
 /*==================Profilbild-Klein-===============================*/
 .user-profile__image-small {
-  width: 2.5rem;
+  width: 6.5rem;
```

Comment:
Die Bilder haben eine unterschiedliche Größe bekommen:
![image](https://github.com/danielkull/capp/assets/119632155/34d51882-56cd-4cd0-b282-15f1e2c1cd5a)

Bitte die Bilder kleiner machen und Einheitlich, ansonsten wird die Karte zu große und es bleibt zu wenige Platz für den Text. Das Bild sollte nur Nice to have sein. Der Content ist das wichtige.

Reply:
Bilder sind jetzt gleich groß